pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -168,6 +169,14 @@ func AddToManager(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	// Watch for provisions
+	if err := c.Watch(&source.Kind{Type: &hivev1.ClusterProvision{}}, &handler.EnqueueRequestForOwner{
+		IsController: true,
+		OwnerType:    &hivev1.ClusterDeployment{},
+	}); err != nil {
+		return err
+	}
```

Comment:
Yes, we still need to watch jobs because there are still jobs used for image resolution.

Reply:
As for the spec changes kicking off a new provision, we had a conversation about that a few weeks ago on slack. You indicated that it would be acceptable to not stop the current install. Any changes to the spec would be reconciled later on the installed cluster.

See https://coreos.slack.com/archives/CE3ETN3J8/p1562697405437400